src/Sylius/Bundle/AdminBundle/TwigComponent/Shipment/ShipType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\TwigComponent\Shipment;
+
+use Sylius\Bundle\AdminBundle\TwigComponent\HookableComponentTrait;
+use Sylius\Component\Core\Model\Shipment;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\ComponentWithFormTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent(name: 'SyliusAdmin.Shipment.ShipType', template: '@SyliusAdmin/Shipment/Component/ship.html.twig')]
```

Comment:
If we want to explicitly define a method, we should stay consistent with all the components

Reply:
I don't understand, can you explain more?

Reply:
It was about `method: 'post'` but since it was just a mistake, this concern is no longer relevant 🕺